docker/load_data_to_lims_db.py

```diff
@@ -74,16 +83,36 @@ def _create_project(self, name, udfs=None, closed=False, researcher=None):
         self.session.add(p)
         return p
 
-    def _create_input_artifact(self, name, samples, container_name, xpos, ypos, original=True):
-        container = t.Container(containerid=self._get_id(t.Container), name=container_name)
-        placemment = t.ContainerPlacement(placementid=self._get_id(t.ContainerPlacement), container=container,
-                                          wellxposition=xpos, wellyposition=ypos)
+    def _create_artifact(self, name, samples, container_name=None, xpos=None, ypos=None, udfs=None, original=True,
+                               reagent_labels=None, qcflag=0):
+        """
+        Create the sqlalchemy Artifact object.
```

Comment:
What is that?

Reply:
This is the manifestation of a Sample in a container represented as an SQL alchemy object.